Off-Top-App/off-top-python
# OT- Merge user data from MySQL and session data from MongoDB

# Task Completed
- Created a get request in flask to get user information from MySQL database.
- Performed a get request already created to get all session data from MONGODB . 
- Merged the data based on each other’s user_id parameter.

# How to test
- Run the file `app.py` located in `off-top-python`. On either postman or web browser visit `http://localhost:5000/merge-user-data`. The output should look like as follow:

![merge user data](https://user-images.githubusercontent.com/53457221/80853818-5b615380-8be8-11ea-9ccd-dca9bd86d3bb.JPG)
